static/js/lezer-parsers/tokens.ts

```diff
@@ -572,7 +576,11 @@ const keywordToToken: Record<number, tokenSpecilizer> = {
             "is": is15,
             "while": while15,
             "call": call15,
-            "with": with15
+            "with": with15,
```

Comment:
Hi, @boryanagoncharenko! I took the liberty to push a fix to the highlighting problems you commented to me on Discord. The problem was that for the tokenizer to use the keywords, they need to be added to the proper dictionary. If you set the variable as a  specialized, you need to add it to the `specialize` dictionary inside the import. The difference between specialize and extend is that with `specialize` every time the parser sees the string it will parse it as the keyword, whereas with extend it will decide weather to use the keyword or plain text based on context.

Reply:
Thanks!